### PR TITLE
Overview: Add Columns

### PR DESCRIPTION
**Pull request by timmyc**
A small PR to get the stage set for #108 - and adding in more blocks to the overview page. For now I have just added two empty cards to show the design working.

image

__To Test__
- Load up the dashboard page and verify the placeholder block/cards look good
- Resize the window and note they collapse to a column at smaller viewports

__Design Questions__
I couldn't find a design with details around when the breakpoint from two columns to a single column should happen, so I went with the 782 breakpoint used elsewhere. Also guessing 20px is the correct padding to have between these columns?

### CODE REVIEW

File: client/dashboard/index.js [hunk 1]
```diff
@@ -19,6 +21,14 @@ export default class Dashboard extends Component {
 				<Header sections={ [ __( 'Dashboard', 'wc-admin' ) ] } />
 				<StorePerformance />
 				<Chart />
+				<div className="woocommerce-dashboard__columns">
+					<div className="woocommerce-dashboard__column">
+						<Card title={ __( 'Top Selling Products', 'wc-admin' ) } />
+					</div>
+					<div className="woocommerce-dashboard__column">
```
Comment (ryelle) [inline client/dashboard/index.js]: Could you pass this className into the Card, or do you see having multiple cards in each column?

File: client/dashboard/style.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+.woocommerce-dashboard__columns {
```
Comment (ryelle) [inline client/dashboard/style.scss]: You can make this file much simpler & not rely on negative margins by using css grid:

```scss
.woocommerce-dashboard__columns {
	display: grid;
	grid-template-columns: 1fr 1fr;
	grid-column-gap: $gutter;

	@include breakpoint( '<782px' ) {
		grid-template-columns: 1fr;
	}
}
```

I would also avoid using px values for any spacing, we should always use our variables - unless it's a CSS magic number situation, where it needs to be absolutely positioned or something, but then it should be commented.

File: client/dashboard/index.js [hunk 3]
```diff
@@ -8,9 +8,11 @@ import { Component, Fragment } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import './style.scss';
 import Header from 'layout/header';
 import StorePerformance from './store-performance';
 import Chart from 'components/chart';
+import Card from '../components/card';
```
Comment (ryelle) [inline client/dashboard/index.js]: We have an alias for `components`, you don't need to use the relative path

**Review by ryelle — CHANGES_REQUESTED** [review]
Framework is good, but I have CSS feedback – if you want to keep the flex layout, we should use our spacing variables & see if we can make it work without negative margins.

File: client/dashboard/index.js (hunk 1 above)
Comment (timmyc) [reply client/dashboard/index.js]: I believe the notion is we will eventually have multiple cards in a column.

File: client/dashboard/style.scss (hunk 2 above)
Comment (timmyc) [reply client/dashboard/style.scss]: TIL thank you.

**Comment by timmyc** [comment]
Thanks for the css ✨ @ryelle - updated code works great. I believe the relative path on the component was due to me having crazy issues with this branch breaking last Thursday, thanks for catching that.

**Comment by LevinMedia** [comment]
@timmyc I added a link to the component grid in p6riRB-3dt-p2 - should address all your design concerns!

**Review by ryelle — APPROVED** [review]
👍 It's correct at 782+; the smallest screen breakpoint isn't totally right, but nothing on the dashboard is & I think we should address that at the app/page level, not just for the columns.

**Pull request merged**